lib/active_fedora/versions_graph.rb

```diff
@@ -2,11 +2,11 @@ module ActiveFedora
   class VersionsGraph < ::RDF::Graph
 
     def all opts={}
-      if opts[:include_auto_save]
-        fedora_versions
-      else
-        fedora_versions.reject { |v| v.label.match("auto") }
+      versions = fedora_versions
+      unless opts[:include_auto_save]
+        versions.reject! { |version| version.label.match("auto") }
       end
+      versions.sort_by { |version| DateTime.parse(version.created) }
```

Comment:
It seems reasonable to raise an exception if Fedora is giving us a non-date here.